go.mod

```diff
@@ -5,3 +5,5 @@ require (
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/stretchr/testify v1.2.2
 )
+
+go 1.13
```

Comment:
My editor kept adding this line, so I left it. Please, let me know if I should revert this.
Thanks